schema/url/standalone-map/plugin-standalone-map-markers.raml

```diff
@@ -34,13 +32,64 @@ types:
       mapElements: StandaloneMapElementInfo[]
 
   # TODO remove once quantile endpoint is merged.
+  NumericBinRange:
```

Comment:
@ryanrdoherty and I were just talking about maybe using HistogramBin instead of BinRange in #240 and i wonder how this change impacts that

Reply:
Not sure.  Might be handy to use the same bin type as the distribution endpoint since then we can write methods to handle both.  Am I wrong that the efficiency of number vs string even when we know it's a number is negligible since this is just the bin definitions and not something for each row in a result?  The plugin can always coerce into the right type for processing and gain the efficiency benefits there.